Admin/ArticleAdmin.php

```diff
@@ -0,0 +1,277 @@
+                ))
+            ->end()
+
+            ->with('Publication', array('class' => 'col-md-4'))
+                ->add('status', ChoiceType::class, array(
+                    'choices' => $this->isGranted('ROLE_ARTICLE_PUBLISH') ? AbstractArticle::getStatuses() : AbstractArticle::getContributorStatus(),
+                    'attr' => array('class' => 'full-width'),
+                    'choices_as_values' => false,
+                ))
+                ->add('publicationStartsAt', DateTimePickerType::class, array(
+                    'format' => 'dd/MM/yyyy HH:mm',
+                    'datepicker_use_button' => false,
+                    'dp_side_by_side' => true,
+                    'dp_language' => 'fr',
+                    'required' => false,
+                ))
+                ->add('publicationEndsAt', DateTimePickerType::class, array(
+                    'format' => 'dd/MM/yyyy HH:mm',
+                    'datepicker_use_button' => false,
+                    'dp_side_by_side' => true,
+                    'dp_language' => 'fr',
+                    'required' => false,
+                ))
+            ->end()
+
+            ->with('Tags', array('class' => 'col-md-6'))
+                ->add('tags', 'sonata_type_model_autocomplete', array(
+                    'required' => false,
+                    'property' => 'name',
+                    'multiple' => true,
+                    'label' => false,
+                    'attr' => array('class' => 'show'),
+                    'callback' => function (AdminInterface $admin, $property, $searchText) {
+                        $datagrid = $admin->getDatagrid();
+                        $datagrid->setValue($property, null, $searchText);
+                        $datagrid->setValue('enabled', null, true);
+                    },
+                ))
+            ->end()
+
+            ->with('Catégories', array('class' => 'col-md-6'))
+                ->add('categories', 'sonata_type_model_autocomplete', array(
+                    'required' => false,
+                    'property' => 'name',
+                    'multiple' => true,
+                    'label' => false,
+                    'attr' => array('class' => 'show'),
+                    'callback' => function (AdminInterface $admin, $property, $searchText) {
+                        $datagrid = $admin->getDatagrid();
+                        $datagrid->setValue($property, null, $searchText);
+                        $datagrid->setValue('enabled', null, true);
+                    },
+                ))
+            ->end()
+
+            ->with('Fragments', array('class' => 'col-md-12'))
+                ->add('fragments', 'sonata_type_collection', array(
+                    'cascade_validation' => true,
+                    'by_reference' => false,
+                    'label' => false,
+                    'pre_bind_data_callback' => function ($value) use ($fragmentClass, $subject) {
```

Comment:
This callback goal is to map the fragment type with the one selected on the list.
You might see article/fragments like Pages/blocks, it's the same logic. 

@bmaziere maybe you can add one screenshot to this PR to show everybody what it's look like, it will take more sense for everybody :)